src/python/WMComponent/DBS3Buffer/DBSUploadPoller.py

```diff
@@ -94,32 +94,21 @@ def uploadWorker(workInput, results, dbsUrl, gzipEncoding=False):
             dbsApi.insertBulkBlock(blockDump=block)
             results.put({'name': name, 'success': "uploaded"})
         except HTTPError as ex:
-            # when we will have new dbs3-client and fine granularity DBS Go server
```

Comment:
Can you please add a link - as comment - to the errors.go code where these errors are defined?

Reply:
done